app/src/main/java/com/doctor/yumyum/presentation/ui/splash/SplashActivity.kt

```diff
@@ -30,18 +30,23 @@ class SplashActivity : BaseActivity<ActivitySplashBinding>(R.layout.activity_spl
             viewModel = viewModel
         }
 
+        window.statusBarColor = getColor(R.color.main_orange)
+
         viewModel.errorState.observe(this) { error ->
             if (error) startActivity(Intent(this, LoginActivity::class.java))
         }
 
         if (viewModel.loginToken.isNullOrBlank()) {
-            startActivity(Intent(this, LoginActivity::class.java))
-        }
-        else {
+            startActivity(Intent(this, LoginActivity::class.java).apply {
+                flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+            })
+            finish()
+        } else {
             CoroutineScope(Dispatchers.IO).launch {
                 viewModel.signIn()
+                startActivity(Intent(this@SplashActivity, MainActivity::class.java))
```

Comment:
꼭 수정해야 하는 부분은 아닌데 같은 함수를 호출하는 경우 같은 형식으로 맞추면 좋을 것 같아요~!
```suggestion
            startActivity(Intent(this@SplashActivity, LoginActivity::class.java).apply {
                flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
            })
            finish()
        } else {
            CoroutineScope(Dispatchers.IO).launch {
                viewModel.signIn()
                startActivity(Intent(this@SplashActivity, MainActivity::class.java))
```

Reply:
이 부분 `flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK`로 플래그를 달면 activity 전환 에니메이션이 좀 다르게 변해서 여기만  `finish`를 사용했습니다!

Reply:
엇 저는 `this` 옆에 액티비티 이름 붙이는 거 말한 건데요...! 👀

Reply:
앗..! 그건 `CoroutineScope` 안에 들어가서 액티비티 이름 넣어준건데! 그럼 다른 브랜치 작업할때 수정해두겠습니당😀

Reply:
아 밑에 부분이 코루틴 스코프 안에 있는 거였네요..! 그럼 지금 상태가 맞는 것 같네용 👍